Apsitvarkom.Models/Public/LocationTitleResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Apsitvarkom.Models.Public;
+
+using FluentValidation;
+using static LocationTitle;
+
+public class LocationTitleResponse
+{
+    public LocationCode Code { get; set; }
+    public string Name { get; set; } = string.Empty;
+}
+
+public class LocationTitleResponseValidator : AbstractValidator<LocationTitleResponse>
+{
+    public LocationTitleResponseValidator()
+    {
+        RuleFor(x => x.Name).NotNull();
```

Comment:
Swagger showed Name as nullable, so I added this. However now it shows it as required, which is a unique case among all the response models, so I'm not too sure if this is needed.